DebugIR.cpp

```diff
@@ -431,8 +432,8 @@ class DIUpdater : public InstVisitor<DIUpdater> {
         DIType *ElDIType = getOrCreateType(ElType);
         DIType *MemType = Builder.createMemberType(
             LexicalBlockFileNode,
-            (T->getStructName().str() + "." +
-             std::to_string(tempNameCounter++)),
+            (!ST->isLiteral()?T->getStructName().str() + "." +
```

Comment:
```suggestion
            (ST->hasName()?T->getStructName().str() + "." +
```

Reply:
Also could you run clang-format on the file once?

Reply:
> Also could you run clang-format on the file once?

No, I didn't, but if it needed i'll do it.

Reply:
> > Also could you run clang-format on the file once?
> 
> No, I didn't, but if it needed i'll do it.

Yes please.

Reply:
>

> > > Also could you run clang-format on the file once?
> > 
> > 
> > No, I didn't, but if it needed i'll do it.
> 
> Yes please.

I ran clang-format, it shows message "no modified files to format"!